package.json

```diff
@@ -33,8 +33,8 @@
     "babel-cli": "^6.24.1",
     "babel-preset-es2015": "^6.24.1"
   },
-  "dependencies": {
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+  "peerDependencies": {
+    "react": "^15.6.1 || ^16.0.0",
+    "react-dom": "^15.6.1 || ^16.0.0"
```

Comment:
Hmm, since this module imports from `react-dom`, shouldn't that stay in the regular `dependencies` section, or is there something special going on here?

Moving `react` to `peerDependencies` seems correct.

Reply:
Having it in dependencies would mean that it could be running a different react-dom version to react-sdk/riot-web and that *could* lead to tests which pass/fail incorrectly due to a version mismatch between runtime and test environment.

Reply:
This is how it's done in react-gemini-scrollback which (successfully) imports both React and DOM

Reply:
Hmm! Okay, I'll assume it makes sense then. 😅